### PR TITLE
New contributors file

### PR DESCRIPTION
**Pull request by gillesbourgeat**

### CODE REVIEW

**Comment by Alban-io** [comment]
:+1:

**Comment by lunika** [comment]
I think this file is useless and can be removed as github provides a complete list of contributors : https://github.com/thelia/thelia/graphs/contributors

**Comment by gillesbourgeat** [comment]
There may be large contributors that do not necessarily make pull requests, they are not on the GitHub contributors page. 
I have a script that uses GitHub API

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ - Chevrier (AnthonyMeedle)
+ - Romain Ducher (air-dex)
+ - Antony Penalver (Soldras)
+ - David Rimbault (Id4v)
+ - Stéphanie Pinet (stephaniepinet)
+ - Damien Foulhoux (Lucanis)
+ - Vincent Lopes-Vicente (lopes-vincent)
+ - (alex63530)
+ - Arnault Pachot (apachot)
+ - Cédric Sibaut (csibaut)
```
Comment (lunika) [inline CONTRIBUTORS.md]: Sibaud...

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ - Jérôme Billiras (bilhackmac)
+ - Emmanuel Nurit (enurit)
+ - (Asturyan)
+ - Etienne Perriere (Mertiozys)
+ - (griotteau)
+ - Alban Baixas (Alban-io)
+ - Christophe (InformatiqueProg)
+ - (jodeq)
+ - (nicolasleon)
+ - Chevrier (AnthonyMeedle)
```
Comment (lunika) [inline CONTRIBUTORS.md]: Anthony

**Comment by lunika** [comment]
As you want, you seem to be the master now.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (gillesbourgeat) [inline CONTRIBUTORS.md]: My bad !

File: CONTRIBUTORS.md (hunk 2 above)
Comment (gillesbourgeat) [inline CONTRIBUTORS.md]: There are no error

**Pull request merged**